libdnf/repo/solvable/Dependency.cpp

```diff
@@ -92,18 +92,7 @@ void SolvableDependency::createLibsolvDependencyAndValidate(Pool *pool, const st
     }
 }
 
-bool SolvableDependency::isRichDependency(std::string dependency) const
+bool SolvableDependency::isRichDependency(std::string &dependency) const
```

Comment:
(**const** std::string & dependency)

Reply:
correct, looks like memory failure in my case :)